updatecli/updatecli.d/known-hosts.yaml

```
@@ -28,15 +28,18 @@ targets:
         spec:
           files:
             - charts/fleet/templates/configmap_known_hosts.yaml
+      # TODO ensure this script is available on each branch
       command: bash .github/scripts/update_known_hosts_configmap.sh
```

Since this is a very simple use case, and only one script is used, you could make this run the script from the `main` branch by using a small git trick: `git show <branch>:</path/to/file>` will retrieve the output of the file as it's in the provided branch.
So maybe something like this could work?
```suggestion
      command: bash <(git show main:.github/scripts/update_known_hosts_configmap.sh)
```

---

That's much simpler than pushing the script to separate branches indeed. Thanks, applied.